Dockerfile

```diff
@@ -31,16 +31,16 @@ RUN apt-get update -y && \
 
 WORKDIR /workspace
```

Comment:
So this workspace is useless now? we better use /root as the default to keep the same with 0.8.4rc2 @Yikun

Reply:
Yes, we need better follow vLLM repo to keep user experience, we can note some on `0.8.4rc2` and `0.8.5rc1` release note to show this.

Reply:
OK. I'm fine.